lightly/loss/swav_loss.py

```diff
@@ -131,10 +132,10 @@ def subloss(self, z: torch.Tensor, q: torch.Tensor):
 
     def forward(
         self,
-        high_resolution_outputs: List[torch.Tensor],
-        low_resolution_outputs: List[torch.Tensor],
-        queue_outputs: List[torch.Tensor] = None,
-    ):
+        high_resolution_outputs: List[Tensor],
+        low_resolution_outputs: List[Tensor],
+        queue_outputs: Union[List[Tensor], None] = None,
```

Comment:
We can also use the new syntax here, we already use it at other places: https://github.com/lightly-ai/lightly/blob/53d1af4c01d674e78d0ba00d38a8239d1c31b7e6/lightly/models/modules/ijepa_timm.py#L103

Reply:
I'd like to keep consistent with the rest of the file. IMO it doesn't make sense to mix the styles. We should decide for one way and then pull it through the whole code base.